modules/system/lang/it/lang.php

```diff
@@ -1,23 +1,61 @@
-<?php
-
-return [
+<?php return [
     'app' => [
         'name' => 'Winter CMS',
         'tagline' => 'Tornare alle origini',
     ],
+    'locale' => [
```

Comment:
Might want to remove the locale array from here, should probably handled from "en" only

Reply:
@mjauvin it's been discussed before and I think we landed on it's fine to have it as it is so that if the fallback locale is set to IT, it still works fine.